docs/projects/t1-2022/thoth-tech-leadership/weekly-todos/week5.md

```diff
@@ -0,0 +1,67 @@
+# Week 5 Todos
+
+## The story so far
+
+By now your entire team should have done a team retrospective. Meetings are hard
+so please try to think of ways to collaborate async with your team members. For
+example, use Miro to collect ideas async and shorten meetings with clear agenda.
+
+Please continue to promote your members efforts using the 'Shoutout!' channel or
+team meetings. It helps to drive engagement and improve team morale.
+
+## Planing
+
+Please organise time for presentation and preparing the materials for the
+Showcase this week.
+
+Remind team to start looking at the Software Requirements Specification in
+preparation for OnTrack Task 6.2.
+
+Continue to build out your Trello boards. For teams who are still struggling
```

Comment:
We will also be starting to provide some specific feedback on your Trello boards this week

Reply:
Addressed in [731dbae](https://github.com/thoth-tech/handbook/pull/49/commits/731dbae5bc4068b9e44eb25e1e1379fcf2ef3bce)